haskell-src-meta/src/Language/Haskell/Meta/Syntax/Translate.hs

```diff
@@ -811,4 +812,241 @@ instance ToDecs (Exts.Decl l) where
 instance ToDecs a => ToDecs [a] where
   toDecs a = concatMap toDecs a
 
+
+-----------------------------------------------------------------------------
+
+-- * Extensions
+
+-- | Returns @Nothing@ when the extension is not supported by template-haskell.
+toExt :: Exts.KnownExtension -> Maybe TH.Extension
```

Comment:
I would call these `toExtension` and `fromExtension` since the types are named *Extension.